storybook/storybook-docs/src/grid.stories.mdx

```diff
@@ -105,3 +109,8 @@ Dat zorgt ervoor dat content op zo veel mogelijk vensterbreedtes en met diverse
 Op dit moment reageert de typografie nog op één breakpoint bij een vensterbreedte van 854 pixels.
 Nu het grid doorontwikkeld is, stemmen we de typografie daar weer op af.
```

Comment:
This is very ambitious since we have yet to do a full layout using the grid. Also, how do we remind ourselves to change this line when the typography has been changed?

Reply:
Bettina is working on this and I also made a prototype while making the narrow Grid variations, albeit with not too much text inside. I expect we’re going to change the current typography breakpoint to one of the two grid breakpoints, tweak the font sizes a bit and that’s it.

Understand the point about tracing back this text. Made a note of it in the appropriate task on our board.